product/views/Host.yaml

```diff
@@ -72,6 +74,7 @@ headers:
 - IPMI Enabled
 - Last Analysis Time
 - Region
+- Auth Status
```

Comment:
Better: Authentication Status (we don't use Auth Status anywhere in our application).

Reply:
Thanks @mzazrivec

I was going for an abbreviated form just to save space in the report, but I really don't have an opinion one way or the other.